nusight2/server.rollup.config.js

```diff
@@ -0,0 +1,94 @@
+// @ts-check
+import { nodeResolve } from '@rollup/plugin-node-resolve'
+import run from '@rollup/plugin-run'
+import typescript from '@rollup/plugin-typescript'
+import url from '@rollup/plugin-url'
+import minimist from 'minimist'
+import * as path from 'path'
+
+const rootDir = __dirname
+
+const isProduction = process.env.BUILD === 'production'
+const isContinuousIntegration = Boolean(process.env.CI)
+
+function getArgs() {
+  const args = minimist(process.argv.slice(2), {
+    boolean: ['run', 'virtual-robots', 'inspect'],
+    string: ['address'],
+  })
+
+  const nonRollupArgs = []
+
+  if (args['virtual-robots']) {
+    nonRollupArgs.push('--virtual-robots')
+  }
+
+  if (args['address']) {
+    nonRollupArgs.push('--address', args['address'])
+  }
+
+  return {
+    run: Boolean(args['run']),
```

Comment:
What's the benefit of this over `!!args['run']` ? I've never seen it done like this before